AppService.Acmebot/Functions/SharedActivity.cs

```diff
@@ -455,9 +461,34 @@ public Task<Certificate> UploadCertificate([ActivityTrigger] (Site, string, byte
         }
 
         [FunctionName(nameof(UpdateSiteBinding))]
-        public Task UpdateSiteBinding([ActivityTrigger] Site site)
+        public async Task UpdateSiteBinding([ActivityTrigger] Site site)
         {
-            return _webSiteManagementClient.WebApps.CreateOrUpdateAsync(site);
+            // _options
+            var Client = _webSiteManagementClient;
+            var builder = new UriBuilder(_environment.ResourceManager);
+            var path = "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}";
+            path = path.Replace("{resourceGroupName}", System.Uri.EscapeDataString(site.ResourceGroup));
+            path = path.Replace("{name}", System.Uri.EscapeDataString(site.Name));
+            path = path.Replace("{subscriptionId}", System.Uri.EscapeDataString(Client.SubscriptionId));
+            builder.Path = path;
+            builder.Query = "?api-version={apiVersion}&skipDnsRegistration=true".Replace("{apiVersion}", Client.ApiVersion);
+
+            var request = new HttpRequestMessage();
+            request.Method = new HttpMethod("PUT");
+            request.RequestUri = builder.Uri;
+
+            var content = JsonConvert.SerializeObject(site, Client.SerializationSettings);
+            request.Content = new StringContent(content, System.Text.Encoding.UTF8);
+            request.Content.Headers.ContentType = System.Net.Http.Headers.MediaTypeHeaderValue.Parse("application/json; charset=utf-8");
+
+            await _credentials.ProcessHttpRequestAsync(request, System.Threading.CancellationToken.None).ConfigureAwait(false);
+
+            var response = await Client.HttpClient.SendAsync(request, System.Threading.CancellationToken.None).ConfigureAwait(false);
+            var statusCode = response.StatusCode;
+
+            if ((int)statusCode != 200 && (int)statusCode != 202) {
+                var ex = new DefaultErrorResponseException(string.Format("Operation returned an invalid status code '{0}'", statusCode));
```

Comment:
Let's have a similar thing done for the certificate upload.

Reply:
@sgryt It imagine it already does so internally, i copy pastaed this code from the Azure SDK